### PR TITLE
WRN-2914: Add a new skin 'custom'

### PR DESCRIPTION
**Pull request by 0x64**
Enact-DCO-1.0-Signed-off-by: Seungcheon Baek (sc.baek@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
New skin 'custom' for customization support.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-2914

### CODE REVIEW

**Comment by jeonghee27** [comment]
Related PR : 
https://github.com/enactjs/sandstone/pull/983
https://github.com/enactjs/sandstone/pull/998
https://github.com/enactjs/sandstone/pull/984

**Review by jeonghee27 — APPROVED** [review]
LGTM

File: styles/color-mixins.less [hunk 1]
```diff
@@ -145,11 +145,11 @@
 }
 .sand-button-selected-bg-color () {
 	background-color: @sand-button-bg-color;
-	filter+_: saturate(1.5) brightness(0.5); // Simulates #3e454d, the spec color
+	filter+_: @sand-button-selected-filter // Simulates #3e454d, the spec color
```
Comment (MikyungKim) [inline styles/color-mixins.less:148]: Please remove the comment

File: styles/color-mixins.less [hunk 2]
```diff
@@ -145,11 +145,11 @@
...
 }
 .sand-button-focus-bg-color () {
 	background-color: @sand-button-bg-color;
-	filter+_: saturate(0) brightness(1.755); // Simulates @sand-spotlight-bg-color
+	filter+_: @sand-button-focus-filter// Simulates @sand-spotlight-bg-color
```
Comment (MikyungKim) [inline styles/color-mixins.less:152]: Please remove the comment

File: styles/color-mixins.less (hunk 1 above)
Comment (MikyungKim) [reply styles/color-mixins.less:148]: Please do the same in https://github.com/enactjs/sandstone/pull/984.

Comment (0x64) [reply styles/color-mixins.less]: ✅

File: styles/color-mixins.less (hunk 2 above)
Comment (0x64) [reply styles/color-mixins.less]: ✅

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**